hello-world/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "hello-world"
+version = "0.0.1"
+dependencies = [
+    "build == 0.10.0"
+]
+
+[project.optional-dependencies]
+test = [
+    "pyspark == 3.3.1",
```

Comment:
Linters, testers and code formatters (pytest, isort, flake8, black, mypy etc.) could be set here as well so that the example project already has them set up. Happy to keep it bare bones like this though.

If it is kept bare bones then the `[pytest..]`, `[tool.isort...]` and `[tool.mypy...]` below are not needed.

Reply:
I have opted for doing these through pre-commit instead, I've added configuration for that but still working on it